sci-geosciences/osgearth/osgearth-2.8.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit cmake-utils
+
+DESCRIPTION="Dynamic map generation toolkit for OpenSceneGraph"
+HOMEPAGE="http://osgearth.org/"
+SRC_URI="https://github.com/gwaldron/osgearth/archive/${P}.tar.gz"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc qt5"
+
+RDEPEND="
+	dev-db/sqlite:3
+	>=dev-games/openscenegraph-3.4[curl,qt5?]
+	dev-libs/protobuf
+	dev-libs/tinyxml
+	net-misc/curl
+	sci-libs/gdal
+	sci-libs/geos
+	sys-libs/zlib[minizip]
+	virtual/opengl
+	x11-libs/libX11
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtopengl:5
+	)"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	doc? ( dev-python/sphinx )"
+
+S=${WORKDIR}/${PN}-${P}
+
+PATCHES=(
+	#"${FILESDIR}"/${PN}-2.6-cmake-options.patch
+	#"${FILESDIR}"/${PN}-2.7-linker.patch
```

Comment:
I don't know about the patches. I know that it builds without the patches, but not with, and the applications seem to work, although omnetpp (the application I needed osgearth for) is giving me trouble, so this could be the cause.

That being said, I'm not sure what the patches do. First, I can't build 2.6 or 2.7 (it seems the GLSL API changed from functions to exposing members directly and other breaks I didn't bother to investigate afterwards). Also, it does seem to link into the X11 libraries as shown by:

`ldd /usr/lib64/osgdb_osgearth_osg.so` (it finds 'libX11-xcb.so.1' among others)

And I can't find something after a quick perusal in osgearth's repo concerning X11_LIBRARIES.

As for the 2 2.6 patches, I didn't investigate, not yet anyway. So, for now, I just removed the patches.